types/src/on_chain_config/timed_features.rs

```diff
@@ -115,7 +118,14 @@ impl TimedFeatureFlag {
                 .with_ymd_and_hms(2025, 3, 11, 17, 0, 0)
                 .unwrap()
                 .with_timezone(&Utc),
-
+            (DisabledCaptureOption, TESTNET) => Los_Angeles
```

Comment:
do we need to modify the time here? @georgemitenkov

Reply:
I think it's fine, testnet is on 1.36 anyway?